tasks/install-discourse.yml

```diff
@@ -43,5 +43,5 @@
   replace:
     path: "{{ discourse_path }}/launcher"
     after:
-    regexp: aufs\|btrfs\|zfs\|overlay\|overlay2
-    replace: aufs|btrfs|zfs|overlay|overlay2|devicemapper
+    regexp: \(aufs\|btrfs\|zfs\|overlay\|overlay2\)
```

Comment:
Can you comment a bit why this is needed? Was the regexp simply incorrect?

Reply:
The regex was matching at each pass, as it wasn't taking into account if devicemapper was already present or not on the line.

Reply:
No idempotence